packages/next/src/elements/Nav/index.client.tsx

```diff
@@ -44,14 +44,14 @@ export const DefaultNavClient: React.FC<{
                 id = `nav-global-${slug}`
               }
 
-              const activeCollection =
-                pathname.startsWith(href) && ['/', undefined].includes(pathname[href.length])
+              const isListView = pathname === href
+              const isActive = pathname.startsWith(href)
 
               const Label = (
                 <span className={`${baseClass}__link-label`}>{getTranslation(label, i18n)}</span>
               )
 
-              if (activeCollection) {
+              if (isListView) {
```

Comment:
We should just move the `pathname === href` check to this line, no real need for the variable.